src/OrchardCore.Modules/OrchardCore.Navigation/PagerShapes.cs

```diff
@@ -214,7 +214,7 @@ public PagerShapes(IStringLocalizer<PagerShapes> localizer)
             var firstPage = Math.Max(1, Page - (numberOfPagesToShow / 2));
             var lastPage = Math.Min(totalPageCount, Page + (numberOfPagesToShow / 2));
 
-            var pageKey = String.IsNullOrEmpty(PagerId) ? "page" : PagerId;
+            var pageKey = String.IsNullOrEmpty(PagerId) ? "pageNum" : PagerId;
```

Comment:
I don't like `pageNum`, but thanks for the fix.

Suggestions:
- `page-number`, because camel casing is not ok in urls
- that's it

Reply:
Okay but need to match the `PagerParameters` field name.

I will try to just use `pagenum` instead, as we use somewhere e.g. `contentitemid` as I remember.